core/src/com/unciv/logic/city/CityConstructions.kt

```diff
@@ -141,6 +144,28 @@ class CityConstructions {
         return result
     }
 
+    fun getProductionMarkup(ruleset: Ruleset): FormattedLine {
+        val currentConstructionSnapshot = currentConstructionFromQueue
```

Comment:
Actually, better yet - getCurrentConstruction().
Then check - is Building, is BaseUnit, etc.

Reply:
EVEN BETTER - remove this entirely! Why do we need production markup? We're on a Civilopedia PR!

Reply:
Ah, I see why you did this.
...
Very well.